publisher/publisher.go

```diff
@@ -251,12 +257,22 @@ func (pub *Impl) singleLogSubmit(
 	ctx context.Context,
 	chain []ct.ASN1Cert,
 	serial string,
-	ctLog *Log) error {
+	ctLog *Log) (*ct.SignedCertificateTimestamp, error) {
 
+	start := time.Now()
```

Comment:
Does the publisher have a fakeclock this should be using instead of `time`?

Reply:
Nope. One could be added but it'd be only used for this latency recording so I don't think it'd be useful.

Reply:
Sounds good. Thanks for sanity checking that